CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 * [CHANGE] **BREAKING CHANGE** Remove `autocomplete_filtering_enabled` feature flag [#3729](https://github.com/grafana/tempo/pull/3729) (@mapno)
 * [CHANGE] Bump opentelemetry-collector to 0.102.1 [#3784](https://github.com/grafana/tempo/pull/3784) (@debasishbsws)
+* [CHANGE] **BREAKING CHANGE** Bump tempo-query dependency to Jaeger 1.58.1 and switch from grpcPlugin to standalone server [#3840](https://github.com/grafana/tempo/issues/3840) (@frzifus)
```

Comment:
How is this breaking change surfaced to users? What do they need to do? 

Does the configuration of tempo-query change?

Reply:
The configuration should be the same. The main change is that Jaeger is no longer part of the resulting image.

Instead of configuring the plugin with Jaeger a user needs to configure Jaeger itself to use the tempo-query proxy as storage backend.

Reply:
Are there any caveats that should be added to the documentation?

Reply:
Mainly that the image no longer serves the tempo connection as a grpc-plugin loaded by jaeger. Instead  its a standalone service which by default listens on `0.0.0.0:7777`. 

The configuration should remain the same, aside from the new address field e.g.:
```diff
+ address: "0.0.0.0:7777"
backend: "my-backend-service"

tls_enabled: true
tls:
  cert_path: "/path/to/server.crt"
  key_path: "/path/to/server.key"
  ca_path: "/path/to/ca.crt"
  server_name: "my.server.name"
  insecure_skip_verify: false
  cipher_suites: "TTLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256"
  min_version: "TLS1.2"

tenant_header_key: "X-Tenant-ID"
services_query_duration: "10s"
```